Source/CesiumRuntime/Public/GlobeAwareDefaultPawn.h

```diff
@@ -98,6 +104,13 @@ class CESIUMRUNTIME_API AGlobeAwareDefaultPawn : public ADefaultPawn {
   UPROPERTY(EditAnywhere, Category = "Cesium", meta = (ClampMin = 0.0))
   double FlyToGranularityDegrees = 0.01;
 
+  /**
+   * A delegate that will be called whenver the pawn finishes flying
+   *
+   */
+  UPROPERTY(BlueprintAssignable, Category = "Cesium");
+  FCompletedFlight onFlightComplete;
```

Comment:
```suggestion
  FCompletedFlight OnFlightComplete;
```

Small nitpick, Unreal's naming convention is usually to start with a capital letter. This is confusing since it conflicts with our typical C++ naming conventions, but we try to follow Unreal convention for variables exposed to blueprint and the Unreal editor (there are probably places where we've forgotten to do so though).